### PR TITLE
Fix issue #368 Adaptive Icons

### PR DESCRIPTION
**Pull request by yvan-sraka**
Introduces the new [Adaptive Icons](https://developer.android.com/preview/features/adaptive-icons.html) feature of Android O!

### CODE REVIEW

**Comment by kz** [comment]
Nice work on the icons! Quite a fan.

File: weechat-android/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta2'
```
Comment (kz) [inline weechat-android/build.gradle]: It looks like the build is failing due to the Gradle version being changed. Will the build still work with the gradle version and `gradle-wrapper.properties` reverted?

Comment (yvan-sraka) [inline weechat-android/build.gradle]: I just pushed a revert to check that :wink:!

**Comment by mhoran** [comment]
Nice. I like it. What do others think?

**Comment by mhoran** [comment]
I bumped the SDK version to 27 and we automatically got adaptive icons. I'm not sure if this is still required? You can check it out in the beta on Play Store. But maybe this makes them look even better? Honestly, I'm not entirely sure...

**Comment by oakkitten** [comment]
we are using an adaptive icon now

**Pull request closed without merging**